src/main/native/windows/file.cc

```diff
@@ -393,6 +393,68 @@ int CreateJunction(const wstring& junction_name, const wstring& junction_target,
   return CreateJunctionResult::kSuccess;
 }
 
+struct DirectoryStatus {
+  enum {
+    kDoesNotExist = 0,
+    kDirectoryEmpty = 1,
+    kDirectoryNotEmpty = 2,
+    kPendingDeleteChildExists = 3,
+  };
+};
+
+// Check the status of a directory, the possible results are
```

Comment:
This description is too generic.

How about: "Check whether the directory and its child elements truly exist, or are marked for deletion."

Reply:
Done.